app/views/contexts/section_summary_context.py

```diff
@@ -21,10 +24,16 @@ def __call__(self, current_location, return_to="section-summary"):
             if isinstance(title_for_location, dict)
             else title_for_location
         )
+
+        page_title = self._schema.get_custom_page_title_for_section(
+            current_location.section_id
+        )
+        page_title = page_title or self._get_safe_page_title(title_for_location)
```

Comment:
Thoughts on having 1 public method `get_page_title()` which can call these 2?

Reply:
I think it's worth it. This `__call__` method is getting busy

Reply:
Done in 670307b7c5f7